README.md

```diff
@@ -212,6 +212,61 @@ From here on you can follow the instructions available at the following sections
 
 All done!
 
+How to run tests
+----------------
+
+If you want to contribute to `ansible-openwisp2` you should run tests
+in your development environment to ensure your changes are not breaking anything.
+
+To do that, proceed with the following steps:
+
+**Step 1**: Clone `ansible-openwisp2`
+
+Clone repository by:
+
+    git clone https://github.com/<your_fork>/ansible-openwisp2.git
+
+**Step 2**: Install docker
+
+If you haven't installed docker yet, you need to install it (example for linux debian/ubuntu systems):
+
+    sudo apt-get install docker.io
+
+**Step 3**: Pull docker image
+
+Pull docker image with this command:
+
+    docker pull registry.gitlab.com/ninuxorg/docker/ansible-<distro>
+
+Substitute `<distro>` with one of these variants:
+
+* `ubuntu:18.04`
+* `ubuntu:16.04`
+* `debian:10`
+* `debian:9`
+
+**Step 4**: Run docker container
+
+To create and start docker container run this command:
+
+    docker run -d --volume=${PWD}/ansible-openwisp2:/etc/ansible/roles/role_under_test:rw --name <name> --privileged registry.gitlab.com/ninuxorg/docker/ansible-<distro> /lib/systemd/systemd
+
+Substitute `<name>` with the desired container name and `<distro>` with the option you have chosen in **Step 3**.
+
+**Step 5** Install requirements in docker container
+
+Now install the requirements with:
+
+    docker exec -t <name> ansible-galaxy install -r /etc/ansible/roles/role_under_test/tests/requirements.yml
+
+**Step 6** Test changes made while development in docker container
```

Comment:
I changed name of Step 6, is it good enough?

before it was: Deploy openwisp2 on docker container

Reply:
'Run tests with:` is enough, simplicity is the key, remove the rest because it's redundant. Documentation needs to be short and straight to the point, most people don't read the whole text anyway.